disnake/ext/commands/slash_core.py

```diff
@@ -156,20 +160,22 @@ class SubCommandGroup(InvokableApplicationCommand):
     def __init__(
         self,
         func: CommandCallback,
+        parent: InvokableSlashCommand,
         *,
         name: LocalizedOptional = None,
         **kwargs,
     ):
         name_loc = Localized._cast(name, False)
         super().__init__(func, name=name_loc.string, **kwargs)
+        self.parent: InvokableSlashCommand = parent
         self.children: Dict[str, SubCommand] = {}
         self.option = Option(
             name=name_loc._upgrade(self.name),
             description="-",
             type=OptionType.sub_command_group,
             options=[],
         )
-        self.qualified_name: str = ""
+        self.qualified_name: str = f"{parent.name} {self.name}"
```

Comment:
```suggestion
        self.qualified_name: str = f"{parent.qualified_name} {self.name}"
```
doesn't matter in practice, but keeps it in line with subcommands.